library/options.inc.php

```diff
@@ -2489,7 +2491,8 @@ function display_layout_tabs_data($formtype, $result1, $result2='') {
 					if ($titlecols > 0) {
 					  disp_end_cell();
 					  $titlecols_esc = htmlspecialchars( $titlecols, ENT_QUOTES);
-					  echo "<td class='label' colspan='$titlecols_esc' ";
+                        $field_id_label='label_'.$group_fields['field_id'];
```

Comment:
Turn off auto-indentation in your editor. Will be important to maintain the spacing of the script you are working in to maintain readability.